cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/kubernetes/CreateDeploymentOperation.java

```diff
@@ -43,8 +43,8 @@ public void execute(Vertx vertx, K8SUtils k8s, Handler<AsyncResult<Void>> handle
                         handler.handle(Future.succeededFuture());
                     }
                     else {
-                        log.error("Deployment creation failed: {}", res.result());
-                        handler.handle(Future.failedFuture((Exception)res.result()));
+                        log.error("Deployment creation failed: {}", res.cause().toString());
```

Comment:
Its so beautiful how one mistake copy-pastes into million places :-)

Reply:
I'm working on that copy+paste thing right now. I think we'll be able to delete quite a few of these operations...